core/src/main/java/smile/regression/OLS.java

```diff
@@ -75,7 +76,7 @@
  * 
  * @author Haifeng Li
  */
-public class OLS implements Regression<double[]>, Serializable {
+public class OLS implements Regression<double[]>, OnlineRegression<double[]>, Serializable {
```

Comment:
OnlineRegression is a subclass of Regression. We don't both of them here.

Reply:
I did this to support better documentation. Using both Regression and OnlineRegression interfaces allows users to identify classes that implement OnlineRegression in the documentation for OnlineRegression, the "All Known Implementing Classes:" section. The same idea holds for the Regression interface. 

What do you think?